### ISSUE TITLE
Add vignette on computation with general likelihoods

### ISSUE THREAD

**Issue by pcarbo** [stephens999/ashr#52]
[ashgen.Rmd.gz](https://github.com/stephens999/ashr/files/725062/ashgen.Rmd.gz)

Thanks to Mengyin.

**Comment by stephens999** [comment]
@mengyin I think it would be good to simplify the use of the general likelihoods
by setting betahat=0 and betahatsd=1 for the user when using likelihoods that require
that. 

In fact, maybe that should be the default: if a likelihood is specified 
and not a t or normal likelihood, and
betahat and bhatsd are missing, set betahat=0 and the sd=1.

So for example
`ash(rep(0, length(y)), 1, lik = lik_pois(y))`
would become just
`ash(lik = lik_pois(y))`

What do you think?

**Comment by mengyin** [comment]
I agree! Having betahat/sebetahat for the general likelihoods indeed looks confusing. It will be much more clear if we can just avoid them.
